s3_deploy/deploy.py

```diff
@@ -108,7 +110,10 @@ def main():
     conf, base_path = config.load_config_file(args.path)
 
     bucket_name = conf['s3_bucket']
-    cache_rules = conf.get('cache_rules', [])
```

Comment:
@psifertex It seems that flake is complaining now because this line was removed. `cache_rule` seems to be needed later on. I haven't looked deeply into to this but is it possible that this line was deleted by accident?